README.md

```diff
@@ -85,9 +85,34 @@ $ docker run -it anmonteiro/lumo
 
 ## Using Lumo
 
-Enter `lumo` at the command line to launch it.
+### Interactive ClojureScript REPL
+
+Enter `lumo` at the command line to launch the ClojureScript REPL.
+
+```clojure
+$ npm init -y && npm i express request request-promise
```

Comment:
maybe this one can become `npm install express...` just for clarity

Reply:
Never a bad idea. Done in 1739274.